### PR TITLE
LegalEntity schema

### PR DESCRIPTION
**Pull request by PavelVesnin**

### CODE REVIEW

File: apps/graphql/lib/graphql_web/middleware/map_get.ex [hunk 1]
```diff
@@ -20,10 +20,11 @@ defmodule GraphQLWeb.Middleware.MapGet do
   end
 
   def call(%{source: source} = resolution, key) do
-    value = case Map.has_key?(source, key) do
-      true -> Map.get(source, key)
-      false -> Map.get(source, to_string(key))
-    end
+    value =
+      case Map.has_key?(source, key) do
```
Comment (kiurchv) [inline apps/graphql/lib/graphql_web/middleware/map_get.ex]: Maybe `if` fits better?

File: apps/graphql/lib/graphql_web/schema.ex [hunk 2]
```diff
@@ -2,16 +2,19 @@ defmodule GraphQLWeb.Schema do
   @moduledoc false
 
   use Absinthe.Schema
-  use Absinthe.Relay.Schema, :modern
+  use Absinthe.Relay.Schema.Notation, :modern
```
Comment (kiurchv) [inline apps/graphql/lib/graphql_web/schema.ex]: `Absinthe.Relay.Schema` requires `Absinthe.Relay.Schema.Notation`, why do not use it?

Comment (kiurchv) [reply apps/graphql/lib/graphql_web/schema.ex]: So, we should use `Absinthe.Relay.Schema.Notation` in the `LegalEntityTypes` module. Instead, we get duplicated types when we use `Absinthe.Relay.Schema` in another module with type definitions.

Comment (PavelVesnin) [reply apps/graphql/lib/graphql_web/schema.ex]: right, it's better option

**Pull request merged**